src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo_logo"
+        />
```

Comment:
The `alt` attribute for the logo image should be more descriptive, reflecting the content or the purpose of the image.